src/eko/evolution_operator/grid.py

```diff
@@ -77,7 +79,8 @@ def __init__(
             interpol_dispatcher=interpol_dispatcher,
         )
         self._threshold_operators = {}
-        self.ome_members = {}
+        self.is_backward = False
```

Comment:
`is_backward` can't be a attribute of `OperatorGrid` since this is a property of an individual `Operator` instead

Reply:
Okay good, I'll fix it.